crypto/keys/ed25519/keys.pb.go

```diff
@@ -6,8 +6,8 @@ package ed25519
 import (
 	crypto_ed25519 "crypto/ed25519"
 	fmt "fmt"
+	proto "github.com/cosmos/gogoproto/proto"
 	_ "github.com/gogo/protobuf/gogoproto"
```

Comment:
@aaronc, @robert-zaremba I am curious why this import stays here, and is not removed or replaced by `cosmos/gogoproto`. This happens in some `*.pb.go` and `*.pulsar.go`.

Reply:
maybe pulsar might be importing it?

Reply:
It was due to outdated files in the buf registry. This is solved :)